### ISSUE TITLE
UI does not reflect enabled elevation

### ISSUE THREAD

**Issue by bojordan** [digitalcreations/MaxTo#350]
For some time now I have been able to run MaxTo with elevation enabled (I believe I have used all versions that enabled it). However, the UI does not reflect this:

![image](https://user-images.githubusercontent.com/626103/53430882-3fd73900-39a4-11e9-9dc3-c89e5e675327.png)

**Comment by vegardlarsen** [comment]
So if you check Task Manager, is MaxTo running elevated?

Under **Details**, you may have to right-click the column headers, click **Select columns** and toggle the **Elevated** checkbox on. Then you should be able to see the various MaxTo processes running. Only 3 of 5 should be running elevated (`MaxTo.Core.exe`, `MaxTo.Companion.x64.exe` and `MaxTo.Companion.x86.exe`).

**Comment by MalteHillmann** [comment]
Same problem here. It says it's running as normal user. Task manager says: 
```
Name - Elevated
MaxTo.Companion.x64.exe - Yes
MaxTo.Companion.x86.exe - Yes
MaxTo.Core.exe - Yes
MaxTo.Server.Exe - No
MaxTo.UI.exe - No
```

Seems to be correct?

**Comment by vegardlarsen** [comment]
It does seem to be correct, yes.

Is your machine in any way locked down using group policy or similar?

**Comment by MalteHillmann** [comment]
> Is your machine in any way locked down using group policy or similar?
Only some privacy modifications by DoNotSpy10. Not in an active directory.

**Comment by vegardlarsen** [comment]
Can you check your registry to see if this key is present, and check its value:

`HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\Policies\System\EnableLUA`

**Comment by MalteHillmann** [comment]
EnableLUA is a REG_DWORD an is set to 1.

**Comment by vegardlarsen** [comment]
OK, so that is the registry key MaxTo uses to determine if User Account Control is enabled on the machine. It appears to be in your case. The processes also seem to be started correctly, but MaxTo itself cannot detect it.

I can't say exactly what causes this, but I will be looking at this closely before 2.0 goes final.

**Comment by MalteHillmann** [comment]
Okay. If you need any help, feel free to contact me.

**Comment by vegardlarsen** [comment]
@MalteHillmann Can you confirm that you are running 2.0.0-rc.1, not any of the beta versions?

**Comment by MalteHillmann** [comment]
@vegardlarsen yes, i'm using "2.0.0-rc1".

**Comment by vegardlarsen** [comment]
@MalteHillmann Can you contact me on the support e-mail so I can have you test a private build?

**Comment by vegardlarsen** [comment]
With @MalteHillmann's help, we have confirmed that our fix works on his machine, and the fix for this issue will be in 2.0.0-rc.2.
